packages/embark/src/lib/utils/accountParser.js

```diff
@@ -65,10 +65,12 @@ class AccountParser {
         logger.warn(`Private key ending with ${accountConfig.privateKey.substr(accountConfig.privateKey.length - 5)} is not a HEX string`);
         return null;
       }
+
+      const key = Buffer.from(accountConfig.privateKey.substr(2), 'hex');
```

Comment:
Would be even better if we checked for the presence of `0x`.
Also, since when does the `fromPrivateKey` method take a buffer? Did we not lock the version of the package?